src/redux/explorer.js

```diff
@@ -422,26 +420,57 @@ export const explorerInitL2 = (network = null) => (dispatch, getState) => {
   if (getState().settings.network === NetworkTypes.mainnet) {
     switch (network) {
       case NetworkTypes.arbitrum:
-        // Start watching arbitrum assets
-        dispatch(arbitrumExplorerInit());
+      case NetworkTypes.polygon:
+        // Start watching arbitrum and polygon assets
+        dispatch(fetchAssetsFromRefraction());
         break;
       case NetworkTypes.optimism:
         // Start watching optimism assets
         dispatch(optimismExplorerInit());
         break;
-      case NetworkTypes.polygon:
-        // Start watching polygon assets
-        dispatch(polygonExplorerInit());
-        break;
       default:
         // Start watching all L2 assets
-        dispatch(arbitrumExplorerInit());
         dispatch(optimismExplorerInit());
-        dispatch(polygonExplorerInit());
```

Comment:
this `default` case is used when `network` is `null` and we want to fetch all the L2 data 
which means we may want to include `dispatch(fetchAssetsFromRefraction());` here as well in lieu of the arbitrum/polygon explorer inits